microsoft-edge/devtools-protocol/0.2/domains/css.md

```diff
@@ -0,0 +1,982 @@
+---
+description: Reference for the CSS Domain. This domain exposes CSS read/write operations. All CSS objects (stylesheets, rules, and styles) have an associated `id` used in subsequent operations on the related object. Each object type has a specific `id` structure, and those are not interchangeable between objects of different kinds. CSS objects can be loaded using the `get*ForNode()` calls (which accept a DOM node id). A client can also keep track of stylesheets via the `styleSheetAdded`/`styleSheetRemoved` events and subsequently load the required stylesheet contents using the `getStyleSheet[Text]()` methods.
+title: CSS Domain - Microsoft Edge DevTools Protocol Version 0.1
```

Comment:
Oops, looks like I need to add a parameter for generating the version here. either than or the protocol.json wasn't versioned correctly